cmake/Platform/Common/Install_common.cmake

```diff
@@ -1131,6 +1131,7 @@ function(ly_setup_o3de_install)
         ${LY_ROOT_FOLDER}/pytest.ini
         ${LY_ROOT_FOLDER}/LICENSE.txt
         ${LY_ROOT_FOLDER}/README.md
+        ${LY_ROOT_FOLDER}/CMakePresets.json
```

Comment:
We also need to include the per platform CMakePresets.json file as well into the SDK layout.

The main CMakePresets.json includes all the per platform CMakePresets.json files which exposes the preset options for each platform.
> https://github.com/o3de/o3de/blob/development/cmake/Platform/Common/CMakePresets.json
https://github.com/o3de/o3de/blob/development/cmake/Platform/Android/CMakePresets.json
https://github.com/o3de/o3de/blob/development/cmake/Platform/Linux/CMakePresets.json
https://github.com/o3de/o3de/blob/development/cmake/Platform/Mac/CMakePresets.json
https://github.com/o3de/o3de/blob/development/cmake/Platform/Windows/CMakePresets.json
https://github.com/o3de/o3de/blob/development/cmake/Platform/iOS/CMakePresets.json

Reply:
Here is how the root level `CMakePresets.json` look:
https://github.com/o3de/o3de/blob/eb9f4f240370e386afbdaac92d6010b4856917d5/CMakePresets.json#L9-L13

Reply:
Interestingly when I did a build install that was apparently making it in there.  Its probably because those folders and files appear in a file_list but the root one does not.

![image](https://github.com/o3de/o3de/assets/70027408/0c4daae6-bf1a-46ef-8730-5753b110c84e)

Reply:
I can't figure out right now HOW those files end up included in the install, but they are.

Reply:
These files are installed due to the `ly_setup_cmake_install` function in [Install_common.cmake](https://github.com/o3de/o3de/blob/7efa78ad9de6301f42eb466482e05024d7dcdffc/cmake/Platform/Common/Install_common.cmake#L581-L589)

Reply:
These files are installed due to the `ly_setup_cmake_install` function in [Install_common.cmake](https://github.com/o3de/o3de/blob/7efa78ad9de6301f42eb466482e05024d7dcdffc/cmake/Platform/Common/Install_common.cmake#L581-L589)